src/core/toga/app.py

```diff
@@ -499,7 +499,10 @@ def visit_homepage(self):
         If the application metadata doesn't define a homepage, this is a no-op.
         """
         if self.home_page is not None:
-            webbrowser.open(self.home_page)
+            if hasattr(self._impl, 'visit_homepage'):
+                self._impl.visit_homepage()
```

Comment:
What's the purpose of this change? If the issue is that `webbrowser.open()` doesn't work on Android, then we should fix that in the Python support library. If it's not possible to fix in a generic way, we should make "open_url" a first-class requirement of the App interface, rather than have this sort of "optional" logic.

Reply:
@freakboy3742 `webbrowser.open` in fact does not work on Android.
I don't know how to fix this in beeware/Python-Android-support - this patching approach is beyond me.
How should we proceed here? Like in `about` where we let the backends implement the appropriate code?
This I could do easily, but I cannot fix `webbrowser.open()` myself.